pkg/services/mssql/dbms_types.go

```diff
@@ -3,8 +3,7 @@ package mssql
 import "github.com/Azure/open-service-broker-azure/pkg/service"
 
 type dbmsProvisioningParams struct {
-	SSLEnforcement string         `json:"sslEnforcement"`
```

Comment:
It should be also removed from the doc.

Reply:
I don't think it was actually IN the docs so this made the code reflect the true state.

Reply:
Oh yes, it is just a invalid parameter.